src/components/signal-viewer/renderer.tsx

```diff
@@ -25,6 +25,13 @@ export default class SignalViewer extends React.Component<Props> {
       });
     });
   }
+  public componentWillUnmount() {
+    Object.keys(this.props.view[key]).map(signal => {
```

Comment:
why do you use `[key]`? `key` is not a variable so don't make it look like one.

Reply:
@domoritz `this.props.view['_signals']` wasn't allowed because of tslint configuration. So had to assign it to a constant.

Reply:
Then find a better name and make it all caps as is standard.